src/sparseml/transformers/text_classification.py

```diff
@@ -464,13 +486,27 @@ def main():
     )
 
     # Preprocessing the datasets
+    if data_args.input_column_names is not None:
+        if "," in data_args.input_column_names:
+            # two input columns
+            columns = data_args.input_column_names
```

Comment:
It looks like the original CLI string is getting assigned here, which would fail the logic below
```suggestion
            columns = data_args.input_column_names.replace(' ', '').split(',')
```

Reply:
going to keep spaces in in case a dataset may have a space in a column name